transforms/transforms.go

```diff
@@ -18,6 +19,48 @@ func TableTransform(transform func(optimus.Row, chan<- optimus.Row) error) optim
 	}
 }
 
+// Explode returns a TransformFunc that takes a row that contains an array
+// in the specified key and returns a duplicate of the row for each value in the array.
+// For example, exploding on "school" for:
+// optimus.Row{"id": "a", "school": []string{"Jones Elementary", "Cedar Middle"}}
+// becomes
+// []optimus.Row{
+//     {"id": "a", "school": "Jones Elementary"},
+//     {"id": "a", "school": "Cedar Middle"},
+// }
+func Explode(keyToExplode string) optimus.TransformFunc {
+	return TableTransform(func(row optimus.Row, out chan<- optimus.Row) error {
+        objToExplode := row[keyToExplode]
+
+        switch reflect.TypeOf(objToExplode).Kind() {
```

Comment:
Is reflection necessary here? Can't we just do a type assertion on the value to be either `[]string` or `[]bson.ObjectId`? (maybe `[]interface{]` and `[]int` too).